shared/commonprops/commonProps.ts

```diff
@@ -10,6 +10,14 @@ export default async function getAllConfigurations(host = undefined) {
   return await CACHED_API.getAllProperties();
 }
 
+export function getConfig(key: string): string {
```

Comment:
please remove this function and use getAllConfiguration, no need for separate function.

Reply:
This is a helper function we've written for future convenience usage

Reply:
Refactored to use getAllConfigurations